framer/BaseClass.coffee

```diff
@@ -19,44 +19,40 @@ class exports.BaseClass extends EventEmitter
 	@define = (propertyName, descriptor) ->
 
 		# See if we need to add this property to the internal properties class
-		if @ isnt BaseClass and descriptor.exportable == true
-			# descriptor.enumerable = true
+		if @ isnt BaseClass
```

Comment:
Style: in CoffeeScript, it's considered better to use `not`, `or` instead of `!` `||`